Applications/Embedding/res/Embedding_split.ini

```diff
@@ -0,0 +1,50 @@
+# Model Section : Model
+[Model]
+Type = Regression	# Network Type : Regression, KNN, NeuralNetwork
+Learning_rate = 0.001 	# Learning Rate
+Epochs = 100		# Epochs
+Optimizer = adam	# Optimizer : sgd (stochastic gradien decent),
```

Comment:
can we update to make optimizer as a new section?

Reply:
Updated :)